OpenRA.Game/Graphics/SpriteLoader.cs

```diff
@@ -21,6 +22,12 @@ public interface ISpriteLoader
 		bool TryParseSprite(Stream s, out ISpriteFrame[] frames);
 	}
 
+	public interface ISpriteLoader2 : ISpriteLoader
```

Comment:
The way how it implemented now allow to use another icn file pair. For example icon2.icn and icon2.map. But if i open 'icon.map' from loader. Then this ability will be broken. There no inforamtion about file name in TryLoadFrames function.